code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -97,6 +113,20 @@
 	max_w_class = ITEM_SIZE_NORMAL
 	can_hold = list(/obj/item/weapon/ore)
 
+/obj/item/weapon/storage/bag/ore/holding
+	name = "satchel of holding"
+	desc = "A revolution in convenience, this satchel allows for infinite ore or produce storage. It's been outfitted with anti-malfunction safety measures."
+	icon_state = "satchel_bspace"
+	max_storage_space = INFINITY
```

Comment:
Sadly there's a maximum cap of what BYOND can allow, I need to find out what it actually is though.

Reply:
@Kurgis I can put a big finite value, just to be safe. I actually took that from Paradise code so I guess if their BYOND can handle `INFINITY` then ours should also be able to do it right?

Reply:
In that case it may of been a legacy issue, which no longer occurs.

Reply:
infinity is just a very high nubmer in byond